chrono-tz/src/lib.rs

```diff
@@ -401,4 +403,9 @@ mod tests {
         #[derive(Hash)]
         struct Foo(Tz);
     }
+
+    #[test]
+    fn test_iana_tzdb_version() {
+        assert_eq!("2023c", IANA_TZDB_VERSION);
```

Comment:
I don't want this test to start failing every time we update the TZ DB. I propose we change this to say that the value should consist of exactly 4 ASCII digits and 1 ASCII letter, instead.